subsys/net/l2/ethernet/lldp/lldp.c

```diff
@@ -127,6 +127,19 @@ static int lldp_send(struct ethernet_lldp *lldp)
 		}
 	}
 
+#if defined(CONFIG_NET_LLDP_END_LLDPDU_TLV_ENABLED)
```

Comment:
Use if (IS_ENABLED(CONFIG_NET_LLDP_END_LLDPDU_TLV_ENABLED)) {...} 

It's nicer.

Reply:
Yes, looks nicer, updated